ci-operator/config/open-cluster-management/grc-ui/open-cluster-management-grc-ui-main.yaml

```diff
@@ -72,6 +76,24 @@ tests:
       IMAGE_TAG: latest
       REGISTRY_ORG: open-cluster-management
     workflow: ocm-ci-image-mirror
+- as: fast-forward
```

Comment:
Can we remove the fast forward for now since we do it as part of a gated test criteria in https://github.com/open-cluster-management/governance-policy-framework?  We can discuss this more as we move forward.